test/thetest.proto

```diff
@@ -582,3 +584,18 @@ message UnrecognizedWithEmbed {
 	optional Embedded embedded = 1 [(gogoproto.embed) = true, (gogoproto.nullable) = false];
 	optional string Field2 = 2;
 }
+
+message ValidatorMessage {
```

Comment:
Done. Added example `proto3` and `proto2` messages that were super useful catching nullable=false issues.